moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -377,44 +294,76 @@ void ServoCalcs::calculateSingleIteration()
   // Calculate this transform to ensure it is available via C++ API
   tf_moveit_to_ee_frame_ = current_state_->getGlobalLinkTransform(parameters_->planning_frame).inverse() *
                            current_state_->getGlobalLinkTransform(parameters_->ee_frame_name);
+}
 
-  have_nonzero_command_ = have_nonzero_twist_stamped_ || have_nonzero_joint_command_;
+void ServoCalcs::publishJointTrajectory(std::unique_ptr<trajectory_msgs::msg::JointTrajectory> joint_trajectory)
```

Comment:
I guess an OutputPipeline comes next? ;)

Reply:
Yes, this is a boundary (preparing and publishing the joint trajectory) I can hopefully break into a set of operations that are connected together.